BUILDING.md

```diff
@@ -130,6 +130,28 @@ From the grpc repository root
 $ bazel build :all
 ```
 
+## cmake: Linux, Using Make
+Run from grpc directory after cloning the repo with --recursive or updating submodules.
+```
+$ mkdir build
+$ cd build
+$ cmake ..
+$ make
+```
+
+If you want to build shared libraries (`.so` files), run `cmake` with `-DBUILD_SHARED_LIBS=ON`.
```

Comment:
because BUILDING.md is mostly for developer build (not for end-users), wouldn't it be better to have the `-DBUILD_SHARED_LIBS=ON` tip in https://github.com/grpc/grpc/blob/master/src/cpp/README.md (which is more focused on how I use gRPC as an end user)?